core/src/mindustry/world/blocks/units/UnitFactory.java

```diff
@@ -70,16 +70,17 @@ public void setBars(){
         bars.add("progress", (UnitFactoryBuild e) -> new Bar("bar.progress", Pal.ammo, e::fraction));
 
         bars.add("units", (UnitFactoryBuild e) ->
-        new Bar(
-            () -> e.unit() == null ? "[lightgray]" + Iconc.cancel :
-                Core.bundle.format("bar.unitcap",
-                    Fonts.getUnicodeStr(e.unit().name),
-                    e.team.data().countType(e.unit()),
-                    Units.getCap(e.team)
-                ),
-            () -> Pal.power,
-            () -> e.unit() == null ? 0f : (float)e.team.data().countType(e.unit()) / Units.getCap(e.team)
-        ));
+            new Bar(
+                () -> e.unit() == null ? "[lightgray]" + Iconc.cancel :
+                    Core.bundle.format("bar.unitcap",
+                        Fonts.getUnicodeStr(e.unit().name),
+                        e.team.data().countType(e.unit()),
+                        Units.getCap(e.team)
+                    ),
+                () -> Pal.power,
+                () -> e.unit() == null ? 0f : (float)e.team.data().countType(e.unit()) / Units.getCap(e.team)
+            )
+        );
     }
```

Comment:
did anything change here, apart from the indentation?

Reply:
~~no, should I revert this~~